src/2048-expert/index.html

```diff
@@ -8,6 +8,7 @@
     <title>2048 Expert</title>
     <link rel="stylesheet" href="../main.css">
     <link rel="stylesheet" href="2048-expert.css">
+    <script type="text/javascript" src="localStorageManager.js" charset="UTF-8"></script>
```

Comment:
Please use `kebab-case` for this file to keep it consistent with other file naming conventions in this repo